michaelmurwayi/Coffee-API
# Ft auth

**What does the PR do?**

- The PR will add permissions and authentication functionality to the system
- It also creates necessary relationships between the models  

**What are the tasks accomplished?**

- Add permission to all viewset
-Add ownership to models and model objects
- Add Token based auth


**How to Manually Test**

- try accessing api resources without auth token
- Create a new user
- Try creating new objects in the models through api website
- Check if you can access items created by other users

**Any Background Information**
None
